proto/regen/ecocredit/v1alpha1/events.proto

```diff
@@ -0,0 +1,75 @@
+syntax = "proto3";
+
+package regen.ecocredit.v1alpha1;
+
+option go_package = "github.com/regen-network/regen-ledger/x/ecocredit";
+
+// EventCreateClass is an event emitted when a credit class is created.
+message EventCreateClass {
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
+
+    // designer is the designer of the credit class.
+    string designer = 2;
+}
+
+// EventCreateBatch is an event emitted when a credit batch is created.
+message EventCreateBatch {
+
+    // class_id is the unique ID of credit class.
+    string class_id = 1;
+
+    // batch_denom is the unique ID of credit batch.
+    string batch_denom = 2;
+
+    // issuer is the issuer of the credit batch.
+    string issuer = 3;
```

Comment:
```suggestion
    // issuer_address is the account address of the issuer of the credit batch.
    string issuer_address = 3;
```
Can we be more explicit that this represents an account address? Or do you think that is unecessary detail?

Reply:
If we don't make it explciit in the field, i think it'd be nice to atleast make it more explicit in the comment.

Reply:
Actually... noticing how we use "issuer" "recipient" "holder" "sender" throughout. I don't mean to push for a change to all those field names, so maybe we can just resolve this within comments (adding an explicit mention to account addresses) and leave the actual field names unchanged.

Reply:
we should update the doc string not the field name IMHO